README.md

```diff
@@ -1805,6 +1815,8 @@ hz.getList('my-distributed-list').then(function (l) {
 });
 ```
 
+Hazelcast List uses `ItemListener` to listen to events that occur when items are added to and removed from the List. See the [Item Listener section](#7523-item-listener) for information on how to create an item listener object and register it.
```

Comment:
Hazelcast List uses `ItemListener` to listen to the events that occur when the items are added to or removed from the List.